src/sentry/ingest/transaction_clusterer/tasks.py

```diff
@@ -0,0 +1,68 @@
+from itertools import islice
+from typing import Any, Sequence
+
+import sentry_sdk
+from django.conf import settings
+
+from sentry import features
+from sentry.models import Project
+from sentry.tasks.base import instrumented_task
+from sentry.utils import metrics
+
+from . import rules
+from .datasource import redis
+from .tree import TreeClusterer
+
+#: Minimum number of children in the URL tree which triggers a merge.
+#: See ``TreeClusterer`` for more information.
+#: NOTE: We could make this configurable through django settings or even per-project in the future.
+#: Minimum number of children in the URL tree which triggers a merge.
+#: See TreeClusterer for more information.
+#: NOTE: We could make this configurable through django settings or even per-project in the future.
+MERGE_THRESHOLD = 100
+
+#: Number of projects to process in one celery task
+#: The number 10 was chosen at random and might still need tweaking.
+PROJECTS_PER_TASK = 10
```

Comment:
Seems quite low , comparing to the number of projects we have. It might be good for the beginning, but I would also pick something like `100` to start with 😄

Reply:
I based this on release monitor, which AFAICT has even smaller batches (one task per org). But your suggestion makes sense to me, let's go with 100 for now.

https://github.com/getsentry/sentry/blob/6c52e4aca106ea6298c0e8469934c02b08d491b2/src/sentry/release_health/tasks.py#L40-L41